src/renderer/lib/directories.ts

```diff
@@ -15,12 +15,15 @@ export function getDefaultRootSlpPath(): string {
   return path.join(root, "Slippi");
 }
 
-export async function findDolphinExecutable(): Promise<string> {
+export async function findDolphinExecutable(
+  type: DolphinType
+): Promise<string> {
   // Make sure the directory actually exists
-  await fs.ensureDir(NETPLAY_PATH);
+  const dolphin_path = path.join(remote.app.getPath("userData"), type);
```

Comment:
We should probably should stick with camelCase for variable names.

```suggestion
  const dolphinPath = path.join(remote.app.getPath("userData"), type);
```